<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>9</number>
    </property>
    <widget class="QWidget" name="page_kuhinja">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QGraphicsView" name="graphicsView_kuhinja">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_kuh_1">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>19</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kuh">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 175, 62);
font: 16pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Kuhinja</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_kuh_2">
      <property name="geometry">
       <rect>
        <x>418</x>
        <y>19</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fr">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>131</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/fridge_1.jpg);



</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pro">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/shop_1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_kupatilo">
     <widget class="QGraphicsView" name="graphicsView_kupatilo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_kup_1">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kup">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>21</y>
        <width>85</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 185, 110);
font: 16pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Kupatilo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_kup_2">
      <property name="geometry">
       <rect>
        <x>418</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_igraonica">
     <widget class="QGraphicsView" name="graphicsView_igraonica">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igraon_1">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_igraon">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>21</y>
        <width>96</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 212, 0);
font: 16pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Igraonica</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igraon_2">
      <property name="geometry">
       <rect>
        <x>418</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igra_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>244</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(191, 64, 64);</string>
      </property>
      <property name="text">
       <string>Igra 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igra_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 210, 22);</string>
      </property>
      <property name="text">
       <string>Igra 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igra_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>164</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 121, 0);</string>
      </property>
      <property name="text">
       <string>Igra 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_igra_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(173, 127, 168);</string>
      </property>
      <property name="text">
       <string>Igra 4</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_spavaca">
     <widget class="QGraphicsView" name="graphicsView_spavaca">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_spav_1">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_spav">
      <property name="geometry">
       <rect>
        <x>276</x>
        <y>21</y>
        <width>98</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
font: 16pt &quot;Ubuntu&quot;;
</string>
      </property>
      <property name="text">
       <string>Spavaca soba</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_spav_2">
      <property name="geometry">
       <rect>
        <x>391</x>
        <y>21</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_prodavn">
     <widget class="QGraphicsView" name="graphicsView_prodavn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 185, 110);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_prod_nazad">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prodavn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 233, 79);
font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Prodavnica</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_friz">
     <widget class="QGraphicsView" name="graphicsView_friz">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 185, 110);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_friz_nazad">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_friz">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 233, 79);
font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Frizider</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_igra_1">
     <widget class="QPushButton" name="pushButton_igra_1_nazad">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_igra_2">
     <widget class="QPushButton" name="pushButton_igra_2_nazad">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_igra_3">
     <widget class="QPushButton" name="pushButton_igra_3_nazad">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_igra_4">
     <widget class="QPushButton" name="pushButton_igra_4_nazad">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
